code/__DEFINES/logging.dm

```diff
@@ -89,7 +89,8 @@
 #define LOG_ENTRY_KEY_SCHEMA_VERSION "s-ver"
 
 /// Log entry header used to mark a file is being reset
-#define LOG_CATEGORY_RESET_FILE_MARKER "{\"LOG ENTRY RESET | SEE NEXT FILE\"}"
+#define LOG_CATEGORY_RESET_FILE_MARKER "{\"LOG FILE RESET -- THIS IS AN ERROR\"}"
+#define LOG_CATEGORY_RESET_FILE_MARKER_READABLE "LOG FILE RESET -- THIS IS AN ERROR"
```

Comment:
Is this used in more than one file? If not keep it there

Reply:
```suggestion
```